index.html

```diff
@@ -14,30 +14,266 @@
+                          <li class="list-less"><p>CANOPY</p></li>
+                          <li class="list"><p>Back End Dev</p></li>
+                          <li class="list"><p>2015</p></li>
+                        </ul>
+                      </div>
+                      <div class="article-container">
+                        <p class="description-item">
+                          A daily selection of privately perosonalized reads; no accounts
+                          or sign-ups required.
+                        </p>
+                      </div>
+                      <div class="tag-container">
+                        <ul class="tag-item">
+                          <li class="tag-list-item"><a class="tag-btn" href="#">html</a></li>
+                          <li class="tag-list-item"><a class="tag-btn" href="#">css</a></li>
+                          <li class="tag-list-item"><a class="tag-btn" href="#">javaScript</a></li>
+                        </ul>
+                      </div>
+                      <div class="buttom-container">
+                        <input type="button" value="See Project" class="project-buttom" />
+                      </div>
+                    </div>
+                </div>
+            </div>            
+        </section>
+
+        <section id="about-myself">
+            <div class="about-content">
+                <div class="about-1">
+                    <div class="about-title-box">
+                        <h2 class="about-header">About myself</h2>
+                    </div>
+                    <div class="about-desc-box">
+                        <p class="description-item">I'm a software developer! I can help you build a product, feature or website look througth some of my work experience! if you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+                    </div>
+                    <div class="Right contact">
+                        <h4 class="media-text">LET’S CONNECT</h4>
+                    </div>
+                    <ul class="Right icons">            
+                        <li><a href="https://twitter.com/asha_cep" target="_blank" class="media-link" rel="noopener"><i class='fab fa-twitter'></i></a></li> 
+                        <li><a href="https://www.linkedin.com/in/asha-vijayan-67179a18" target="_blank" class="media-link" rel="noopener"><i class='fab fa-linkedin'></i></a></li>
+                        <li><a href="https://medium.com/" target="_blank" class="media-link" rel="noopener"><i class='fab fa-medium-m'></i></a></li>
+                        <li><a href="https://github.com/asharanjith" target="_blank" class="media-link" rel="noopener"><i class='fab fa-github'></i></a></li>
+                        <li><a href="https://angel.co/asha-sethu-vijayan" target="_blank" class="media-link" rel="noopener"><i class='fab fa-angellist'></i></a></li>
+                    </ul>
```

Comment:
- [ ] Kindly try to follow the Figma design as much as possible, this includes `font-family`, text `color`, and the alignment of the social icons

## Your design:
![image](https://user-images.githubusercontent.com/50721479/192101708-2febe26d-445f-4fd9-9edb-aa0790914164.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/192101724-e271788b-f402-4f9f-8726-f7e96822b522.png)